graphql/src/main/java/com/linecorp/armeria/server/graphql/DefaultGraphqlService.java

```diff
@@ -53,17 +50,20 @@ final class DefaultGraphqlService extends AbstractGraphqlService implements Grap
 
     private final boolean useBlockingTaskExecutor;
 
+    private final GraphqlErrorHandler errorsHandler;
+
     DefaultGraphqlService(GraphQL graphQL, DataLoaderRegistry dataLoaderRegistry,
-                          boolean useBlockingTaskExecutor) {
+                          boolean useBlockingTaskExecutor, GraphqlErrorHandler errorsHandler) {
         this.graphQL = requireNonNull(graphQL, "graphQL");
         this.dataLoaderRegistry = requireNonNull(dataLoaderRegistry, "dataLoaderRegistry");
         this.useBlockingTaskExecutor = useBlockingTaskExecutor;
+        this.errorsHandler = errorsHandler;
     }
 
     @Override
     protected HttpResponse executeGraphql(ServiceRequestContext ctx, GraphqlRequest req) throws Exception {
-        final MediaType produceType = GraphqlUtil.produceType(ctx.request().headers());
-        if (produceType == null) {
+        final MediaType negotiatedProduceType = GraphqlUtil.produceType(ctx.request().headers());
```

Comment:
Is there any reason to change this? 🤔

Reply:
I changed it to the same parameter name of the `GraphqlErrorHandler#handler`.
https://github.com/line/armeria/blob/58f402c40389a0f18f9e26725fd79d8c4179cb7e/graphql/src/main/java/com/linecorp/armeria/server/graphql/DefaultGraphqlErrorHandler.java#L37-L42

I named the variable negatedProduceType by referring to the JavaDoc of GraphqlUtil.
https://github.com/line/armeria/blob/c087d693a24fa3f0800aaed68c7c5d3657026911/graphql-protocol/src/main/java/com/linecorp/armeria/internal/server/graphql/protocol/GraphqlUtil.java#L34-L35

Reply:
I changed the logic to not passing the `MediaType` to the `handler` and changed the variable name back to the original one.

Reply:
I recommended it because it's clearer when we pass it to the `GraphqlErrorHandler.handle()`. 😉